server/backend/testspg/conftest.py

```diff
@@ -160,3 +176,46 @@ async def _get_current_requester_by_token(access_token: str) -> Requester:
         return requester
 
     return _get_current_requester_by_token
+
+
+@pytest.fixture(scope="function")
+def create_test_user_role_assignments(
```

Comment:
just wondering could we combine this one and the one below (I created before)? feel they almost do the same thing, thanks!!

Reply:
oh sorry please ignore me, I see what you're trying to do here, all good